GVRf/Framework/jni/engine/importer/assimp_importer.h

```diff
@@ -44,7 +66,8 @@ class AssimpImporter: public HybridObject {
     unsigned int getNumberOfMeshes() {
         return assimp_importer_->GetScene()->mNumMeshes;
     }
-
+    void sceneRecursion(aiNode* node, const aiScene* aScene, std::shared_ptr<Scene> scnPtr, JNIEnv * env, jobject obj, jobject bitmap, aiMatrix4x4 accumulatedTransformation);
```

Comment:
That’s just the jni calling convention – I think that would be the this pointer in a non-static method.

From: jason2kim [mailto:notifications@github.com]
Sent: Tuesday, April 21, 2015 2:42 PM
To: Samsung/GearVRf
Cc: Jon Shemitz
Subject: Re: [GearVRf] Support for loading the full hierarchy in a model in case it exists (#18)


In GVRf/Framework/jni/engine/importer/assimp_importer.h<https://github.com/Samsung/GearVRf/pull/18#discussion_r28825365>:

> @@ -44,7 +66,8 @@ class AssimpImporter: public HybridObject {

>      unsigned int getNumberOfMeshes() {

>          return assimp_importer_->GetScene()->mNumMeshes;

>      }

> -

> +    void sceneRecursion(aiNode* node, const aiScene* aScene, std::shared_ptr<Scene> scnPtr, JNIEnv * env, jobject obj, jobject bitmap, aiMatrix4x4 accumulatedTransformation);

The parameter jobject obj is not used, but is it here for future use?

—
Reply to this email directly or view it on GitHub<https://github.com/Samsung/GearVRf/pull/18/files#r28825365>.